src/components/table/examples/table-selectable-row.tsx

```diff
@@ -0,0 +1,38 @@
+import { Component, h, State } from '@stencil/core';
+import { Column } from '../table.types';
+import { persons, columns, Person } from './persons';
+
+@Component({
+    tag: 'limel-example-table-selectable-row',
+    styleUrl: 'table.scss',
+    shadow: true,
+})
+export class TableExampleSelectableRow {
```

Comment:
We should rename this example since the functionality now is called "Activate" instead of "Select"